src/widgets/histogram/chart.js

```diff
@@ -3,13 +3,13 @@ var _ = require('underscore');
 var d3 = require('d3');
 var d3Interpolate = require('d3-interpolate');
 var cdb = require('cartodb.js');
-var tinycolor = require('tinycolor2');
```

Comment:
Is it used anywhere else in the project? If not we could remove it from the `package.json`

Reply:
I think it is not used anywhere else, I searched and there were only two occurrences: https://github.com/CartoDB/deep-insights.js/search?utf8=%E2%9C%93&q=tinycolor&type=

So I am going to remove it from package.json